src/tokens/shasta.ts

```diff
@@ -0,0 +1,13 @@
+import { ChainId } from '../chainIds';
+import { IToken } from '../interfaces';
+
+export const tokens: IToken[] = [
+  {
+    name: 'TRX',
+    address: '0x0000000000000000000000000000000000000000',
+    chainId: ChainId.Shasta,
+    enabled: true,
+    decimals: 6,
+    symbol: 'TRX',
+  },
```

Comment:
add pls this token
`{
    name: 'USDT',
    address: 'TSsjPxRVQxN5CKk59Gfxyme5vhgp3ymSKq',
    chainId: ChainId.Shasta,
    enabled: true,
    decimals: 18,
    symbol: 'USDT',
  }`

Reply:
Done!